src/Core/src/Platform/Android/ButtonExtensions.cs

```diff
@@ -1,72 +1,47 @@
-﻿using Android.Content.Res;
+﻿using System;
+using System.Diagnostics.CodeAnalysis;
+using System.Runtime.CompilerServices;
+using Android.Content;
+using Android.Content.Res;
 using Android.Graphics.Drawables;
-using Android.Graphics.Drawables.Shapes;
 using Android.Widget;
 using Google.Android.Material.Button;
-using Java.Util;
 using Microsoft.Maui.Graphics;
+using static Android.Icu.Text.ListFormatter;
+using static Android.Provider.MediaStore.Audio;
+using AColor = Android.Graphics.Color;
+using R = Android.Resource;
 
 namespace Microsoft.Maui.Platform
 {
 	public static class ButtonExtensions
 	{
-		const int BackgroundDrawableId = 999;
+		const int MauiBackgroundDrawableId = 1001;
+		const int MauiStrokeDrawableId = 1002;
 
-		public static void UpdateBackground(this MaterialButton platformView, IButton button)
-		{
-			platformView.UpdateBorderDrawable(button);
-		}
-
-		public static void UpdateStrokeColor(this MaterialButton platformView, IButton button)
-		{
-			if (platformView.Background is RippleDrawable background)
-			{
-				var gradientDrawable = background.FindDrawableByLayerId(BackgroundDrawableId);
-
-				if (gradientDrawable is not null)
-				{
-					platformView.UpdateBorderDrawable(button);
-					return;
-				}
-			}
+		// Default obtained by running on Android 34 and inspecting the default drawable
+		const int DefaultCornerRadius = 4;
```

Comment:
Are these values correct for all supported API levels?

Reply:
I copied from 34, so maybe... I was hoping consistency is better but we can check later maybe so we can get this PR in.

Reply:
This is material, so most likely the values are the same across all OS versions - but I have not checked.